code-challenges.test.js

```diff
@@ -28,24 +30,80 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
+
+// describe("divisibleBy3", ()=>{
+//     it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", ()=>{
+//         expect(divisibleBy3({ number: 15 })).toEqual("15 is divisible by three")
+// })
+// })
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+
+// describe("divisibleBy3", ()=>{
+//     it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", ()=>{
+//         expect(divisibleBy3({ number: 0 })).toEqual("0 is divisible by three")
+// })
+// })
+// const object3 = { number: -7 }
+
+// describe("divisibleBy3", ()=>{
+//     it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", ()=>{
+//         expect(divisibleBy3({ number: -7 })).toEqual("-7 is not divisible by three")
+// })
+// })
```

Comment:
For best practice, we want 1 test for 1 function, meaning you want 1 describe, 1 it, and then however many expect statements depending on how many test variables are provided.